### PR TITLE
MARKET-135 added demo

### PR DESCRIPTION
**Pull request by ivy-lmu**

### CODE REVIEW

File: tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+					<p:spacer />
+
+				</h:panelGrid>
+				<br />
+				<p:orderList value="#{data.matches}" var="match" responsive="true"
+					style="width:400px" itemValue="#{match}" converter="ivy.ListItem"
+					id="personsList" controlsLocation="right">
+					<f:facet name="caption">Matches</f:facet>
+					<p:column>#{match}</p:column>
+				</p:orderList>
```
Comment (ivy-rew) [inline tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml:36]: just a minor, but can we make the entries look sexier by keeping the line breaks
Selection_1006
?

Comment (ivy-rew) [reply tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml:36]: Maybe we can build something like this: with columns?
https://www.primefaces.org/showcase/ui/data/orderList.xhtml?jfwid=1bfd1

File: tel-search-connector/processes/PhoneDirectory.mod [hunk 2]
```diff
@@ -23,6 +23,7 @@ Py0 f0 outParamDecl '<java.util.List<String> matches> result;' #txt
 Py0 f0 outParamTable 'result.matches=in.matches;
 ' #txt
 Py0 f0 callSignature search(String) #txt
+Py0 f0 @CG|tags connector #txt
 Py0 f0 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (ivy-rew) [inline tel-search-connector/processes/PhoneDirectory.mod:27]: just saw that this process has yet no description... therefore we don't see any when picking connectors.
via right click -> inscribe process -> description you can provide one that will be shown in the connector activity selection dialog
Selection_1008
Selection_1007

**Review by ivy-rew — APPROVED** [review]
looks good: some minor comments which you may provide as follow up PRs or as you choose in addition to this already opened PR.

File: tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+					<p:inputText value="#{data.search}" id="Search" required="false"
+						style="float:left" />
+
+					<p:commandButton id="Find" actionListener="#{logic.find}"
+						ajax="false" value="Find" update="form" icon="ui-icon-search" />
+					<p:spacer />
+
+				</h:panelGrid>
+				<br />
+				<p:orderList value="#{data.matches}" var="match" responsive="true"
```
Comment (ivy-rew) [inline tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml:31]: On the UI I'd streamline the wording with the official tel.search.ch ... therefore I'd rather speak of 'Entries' than 'Matches
Selection_1009
'

File: tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml (hunk 1 above)
Comment (ivy-rew) [reply tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml:36]: I'd assume that we must just replace '\n' with html output <br/>

Comment (ivy-lmu) [reply tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml:36]: replacing \n with <br/> does not seem to work. do I need to add anything else?

Comment (ivy-rew) [reply tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml:36]: Ok. So I'd leave it as is. Maybe @ivy-lli has a good idea how to build a nicer list output. We can have a look at it tomorrow in Zug.

**Review by ivy-rew — APPROVED** [review]

File: tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml (hunk 1 above)
Comment (ivy-lli) [reply tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml:36]: As far as I understand this connector, you can only search for addresses and then they are listed below right?
I think the control's wouldn't be needed in this case right? So I would hide them.

To fix the line breaks, I'm not sure how this work with this widget. I'm not very used to it. But maybe you could simply wrap the match inside a `<pre>` element:
`<p:column><pre>#{match}</pre></p:column>`

Comment (ivy-lmu) [reply tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml:36]: thanks @ivy-lli with <pre> it works :)

**Pull request merged**